data/map.txt

```diff
@@ -21068,6 +21144,9 @@ system "Sagittarius A*"
 	pos 112 22
 	government Uninhabited
 	habitable 10000
+	hazard "Black Hole Gravity" 1
+	hazard "Black Hole Event Horizon" 1
+	hazard "Black Hole Event Horizon" 10
```

Comment:
Looks like a copy paste moment.
```suggestion
	hazard "Black Hole Accretion Disk" 1
```
If you want to reduce the DPS of the accretion disk then I'd reduce the damage value instead of increasing the spawn time, as that is more deterministic than changing these values here, which result in randomized spawn times with this just being the average time between spawns.